plugins/fcp_can_c/tests/006_rpc_support/test.fcp

```
@@ -0,0 +1,22 @@
+version: "3"
+
+struct Msg {
+    s1 @ 0: u8,
+    s2 @ 1: u16,
+}
+
+impl can for Msg {
+    id: 10,
+    device: "ecu",
+}
+
+service SensorService @0 {
+    method RequestState(SensorReq) @0 returns SensorInformation,
```

oh boy! ok, we need to fail if these are not defined.